### ISSUE TITLE
Highlight target replicates as well as sample neighbours

### ISSUE THREAD

**Issue by starcalibre** [microscopium/microscopium-ui#55]
When a point on the plot is clicked, the point itself is highlighted along with it's neighboring points. Any replicate (or in the case of controls, all other controls) samples should also be highlighted. We'll need to decide how these should be coded -- by colour? A differently shaped point?

With respect to implementing this, this can be done in one of two ways:
1) Send a Mongo query that finds all samples with the same target as the point clicked.
2) Add a new field to each sample, say ``replicate_samples`` that stores these IDs.

**Comment by jni** [comment]
Indeed, this is not a big technical challenge but rather a quite difficult *design* challenge. To make referencing of the two choices we face easier:

**A - Visual marking of replicates**
 - [ ] 1. colour
 - [ ] 2. shape
 - [ ] 3. both
 - [ ] 4. line connecting dots

**B - Querying/caching**
 - [ ] 1. Query mongo by treatment
 - [ ] 2. Store an additional entry in mongo docs

**Comment by jni** [comment]
I think it's a good idea to keep our mongo docs as simple as possible so I slightly favour B1 over B2, but I'm not sure about the performance impact of that.
